documentation/source/users/arkane/input.rst

```diff
@@ -94,6 +94,7 @@ Model Chemistry                                  AEC   BC   SOC  Freq Scale Supp
 ``'B3LYP/6-311+G(3df,2p)'``                       v    v    v    v (0.967)  H, C, N, O, P, S
 ``'B3LYP/6-31G**'``                               v    v         v (0.961)  H, C, O, S
 ``'MRCI+Davidson/aug-cc-pV(T+d)Z'``               v         v               H, C, N, O, S
+``'wb97x-d/aug-cc-pvtz'``                         v                         H, C, N, O
```

Comment:
It seems you do include spin-orbit corrections; could you check that column as well?

Reply:
Thanks for the catch. Fixed it!